content/en/seachat/seachat-manual/02-create-agent/03-advanced-settings/03-agent-memory.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Agent Memory "
-description: "SeaChat Memory tracks customer profiles to ensure relevant conversations and assist in expected outcomes."
+title: "Extraction"
+description: "SeaChat Extraction tracks customer profiles to ensure relevant conversations and assist in expected outcomes."
```

Comment:
SeaChat Context Extraction tracks customer conversations and extract information from the conversation utterances.